install_cluster/tasks/main.yml

```diff
@@ -3,27 +3,32 @@
 
 # TODO: Validate cluster settings
 
+- name: install cluster from cluster.yml
```

Comment:
It this is purely SNO perhaps put SNO in the name and file name. `cluster.yml` is somewhat ambiguous